app/src/main/java/com/github/scribeWizTeam/scribewiz/models/ChallengeSubmissionModel.kt

```diff
@@ -14,7 +14,7 @@ data class ChallengeSubmissionModel (
     override val id: String = "",
     val date: Date? = Date(),
     val recordId : String? = "",
-    val challengeId : String? = "",
+    val challengeId : String = "",
```

Comment:
How come IDs are now not nullable?

Reply:
ID's should not be null and always have a "valid" value. It makes the logic simpler, avoiding the null-check everytime we use the id by giving them a default value.